### DISCUSSION TITLE
Range filter in my table

### DISCUSSION THREAD

**Discussion by brunowilliang** [pocketbase/pocketbase#1738] [Q&A]
I have a "day" column and I need to do a range filter to select it.
ex: 01/01/2023 to 02/01/2023.

how could i do that?

![Screenshot 2023-02-01 at 05 56 07](https://user-images.githubusercontent.com/41432581/215996761-e64befad-cad7-4258-a44c-0f5cf729896f.png)

**Comment by ganigeorgiev** [comment]
Since you are using a plain `text` field, I'm not sure if this can be achieved with this date format.

If you reverse the format and change it to `Y/m/d` then you should be able to use the `>/>=` and `</<=` operators, eg:
```js
day >= '2023/01/01' && day <= '2023/01/02'
```
